### ISSUE TITLE
filament has no items

### ISSUE THREAD

**Issue by blalor** [agrif/OctoPrint-InfluxDB#6]
Seeing this in the octoprint log:

```
2020-03-22 15:17:01,182 - octoprint.plugin - ERROR - Error while calling plugin influxdb
Traceback (most recent call last):
  File "/opt/octoprint-venv/lib/python3.8/site-packages/OctoPrint-1.4.0-py3.8.egg/octoprint/plugin/__init__.py", line 224, in call_plugin
    result = getattr(plugin, method)(*args, **kwargs)
  File "/opt/octoprint-venv/lib/python3.8/site-packages/octoprint_influxdb/__init__.py", line 229, in on_event
    for filname, filval in job.get('filament', {}).items():
AttributeError: 'NoneType' object has no attribute 'items'
2020-03-22 15:17:01,426 - octoprint.plugin - ERROR - Error while calling plugin influxdb
Traceback (most recent call last):
  File "/opt/octoprint-venv/lib/python3.8/site-packages/OctoPrint-1.4.0-py3.8.egg/octoprint/plugin/__init__.py", line 224, in call_plugin
    result = getattr(plugin, method)(*args, **kwargs)
  File "/opt/octoprint-venv/lib/python3.8/site-packages/octoprint_influxdb/__init__.py", line 229, in on_event
    for filname, filval in job.get('filament', {}).items():
AttributeError: 'NoneType' object has no attribute 'items'
```

Octoprint 1.4.0, influxdb plugin 1.2.0

**Comment by agrif** [comment]
Ah, looks like maybe 1.4.0 changed something. This shouldn't be too hard to fix -- I'll try to check it out tonight.

**Comment by agrif** [comment]
I've just tagged and released 1.2.1. It should show up in the plugin updater automatically. Give it a whirl, and feel free to re-open this issue if this doesn't fix it. Thanks for the report!

**Comment by blalor** [comment]
Looks good! Thanks!

-- 
Brian Lalor (he/him)
blalor@bravo5.org
